src/framework/CMakeLists.txt

```diff
@@ -112,6 +105,19 @@ if (MUSE_MODULE_UPDATE)
     add_subdirectory(update)
 endif()
 
+# Tests
+if (MUSE_BUILD_UNIT_TESTS)
+    enable_testing()
+    message(STATUS "Enabled testing")
+
+    define_property(TARGET PROPERTY OUTPUT_XML
+        BRIEF_DOCS "List XML files outputted by google test."
+        FULL_DOCS "List XML files outputted by google test."
+    )
```

Comment:
Not 100% on topic, but I've often wondered, what does this actually do?

Reply:
I'm not sure exactly, but I think this is necessary for the integration of gtest and QtCreator  (QtCreator parse output from gtest)